src/lib/locals.js

```diff
@@ -7,6 +7,9 @@ module.exports = {
     );
     res.locals.ga4Disabled = req.app.get("APP.GTM.GA4_DISABLED");
     res.locals.uaDisabled = req.app.get("APP.GTM.UA_DISABLED");
+    res.locals.analyticsDataSensitive = req.app.get(
+      "APP.GTM.ANALYTICS_DATA_SENSITIVE",
```

Comment:
Think we probably need to start documenting the env vars that CE requires in the README or something but that might be for another time...

Reply:
I was thinking this, current pattern seems to be documenting them within the CRIs rather than here. I assume this is because the variables are declared in the CRIs rather than here, but it would make sense to have a section in this README to outline the common variables and what they're for. I'll add it in as we've got the opportunity and it makes sense to be included with this change

Reply:
Amazing thanks Andy!